google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITSpannerOptionsTest.java

```diff
@@ -106,29 +101,26 @@ public void testCloseAllThreadsWhenClosingSpanner() throws InterruptedException
       }
       // Check the number of threads after the query. Doing a request should initialize a thread
       // pool for the underlying SpannerClient.
-      assertThat(
-          getNumberOfThreadsWithName(SPANNER_THREAD_NAME),
-          is(equalTo(DEFAULT_NUM_CHANNELS * NUM_THREADS_PER_CHANNEL + baseThreadCount)));
+      assertThat(getNumberOfThreadsWithName(SPANNER_THREAD_NAME))
+          .isEqualTo(DEFAULT_NUM_CHANNELS * NUM_THREADS_PER_CHANNEL + baseThreadCount);
```

Comment:
Why here shouldn't be `isAtMost`?

Reply:
These threads should all be present as long as the `Spanner` instance is opened and being used. Closing it would normally bring the number of threads back to the `baseThreadCount`, but it can happen that additional threads are (temporarily) removed after closing it.